File: Unit 1/Problem2.java
import java.util.Scanner;

public class Problem2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n, a = 0, b = 1, temp;

        System.out.print("Enter the number of terms: ");
        n = scanner.nextInt();

        for(int i = 1; i <= n; i++) {
            System.out.print(a + " ");
            temp = a + b;
            a = b;
            b = temp;
        }

    }
}

File: Unit 1/Problem2.pas
program Problem2;
var
    n, i: integer;
    a, b, temp: longint;

begin
    write('Enter number of terms: ');
        readln(n);

        a := 0;
        b := 1;

    for i := 1 to n do
    begin   
        write(a, ' ');
        temp := a + b;
        a := b;
        b := temp;
    end;
end.